guides/contributor/CLA.md

```diff
@@ -0,0 +1,88 @@
+<!-- cspell:ignore Docu heptio gitdm -->
+# The Contributor License Agreement
+
+The [Cloud Native Computing Foundation](https://www.cncf.io) (CNCF) defines
+the legal status of the contributed code in two different types of _Contributor License Agreements_
+(CLAs), [individual contributors](https://github.com/cncf/cla/blob/master/individual-cla.pdf) and [corporations](https://github.com/cncf/cla/blob/master/corporate-cla.pdf).
+
+OpenTelemetry can only accept original source code from CLA signatories.
+
+It is important to read and understand this legal agreement.
+
+## How do I sign?
+
+After creating your first Pull Request, the linux-foundation-easycla bot will respond with information regarding your CLA status along with a link to sign the CLA.
+
+<img width="1065" alt="EasyCLA bot" src="https://user-images.githubusercontent.com/69111235/152226443-f6fe61ee-0e92-46c5-b6ea-c0deb718a585.png">
+
+#### 1. If you are signing up as a corporate contributor, ensure that you have linked your corporate email address to your GitHub profile (it doesn't have to be your primary email address for GitHub) or else it can lead to issues with the CLA system.
```

Comment:
it might be worth calling out that signing up as a corporate contributor might require some internal steps that need to be completed as well, if it exists their company's OSPO is the right point of contact for that (and also the right point of contact if everything goes south with the corporate contributor license agreement, which I saw happening a few times already).

A common requirement by corporations might also be that contributions are done using the corporate email address, if wanted I can write a few words on how one can setup their git config to make sure the right email address is used.

Reply:
will create a follow up issue/PR for that